### PR TITLE
DCD-1223: support postgres 12

### PR DESCRIPTION
**Pull request by bpar476**
Adds option to deploy Postgres or Aurora Postgres with Postgres 12 (which is now supported by Bitbucket)

### CODE REVIEW

File: templates/quickstart-database-for-atlassian-services.yaml [hunk 1]
```diff
@@ -169,10 +170,12 @@ Mappings:
       "9": "9.6"
       "10": "10"
       "11": "11"
+      "12": "12"
     AuroraPostgreSQL:
       "9": "9.6.12"
       "10": "10.7"
```
Comment (nanux) [inline templates/quickstart-database-for-atlassian-services.yaml]: Based on this output, should we update the version 9 and 10 as well? I know it is region specific but there is probably a newer version of 10 and 9 available in all regions.

```
aws rds describe-db-engine-versions --engine aurora-postgresql --query '*[].[EngineVersion]' --output text --region ap-southeast-2        
9.6.3
9.6.6
9.6.8
9.6.9
9.6.11
9.6.12
9.6.16
9.6.17
9.6.18
9.6.19
10.5
10.6
10.7
10.11
10.12
10.12
10.13
10.14
11.6
11.7
11.8
11.9
12.4
```

Comment (bpar476) [reply templates/quickstart-database-for-atlassian-services.yaml]: Oh yeah we should I must have misread the output and thought 9 and 10 were on latest already. No worries :)

**Comment by bpar476** [comment]
I'm going to do a test deployment that deploys postgres 12 on Aurora, will come back with results.

**Comment by bpar476** [comment]
Deployment of Aurora with PG 12:
image

**Comment by bpar476** [comment]
And now standard RDS postgres 12 
image

**Review by nghazali — APPROVED** [review]

**Review by nanux — APPROVED** [review]

**Pull request merged**